### ISSUE TITLE
Launch and manage a workflow outside of a Study

### ISSUE THREAD

**Issue by danfunk** [sartography/cr-connect#204]
We should make sure it is possible to have a workflow that can execute outside of a study.  
Possible cases for this might include running through a price check on equipement, or checking to see what might be required of a data security plan. 

Backend work on this ticket is complete, still needs frontend work per comments below.

Moving to the Icebox for now since the initial need for this ticket, launching a non-study specific Grant Pricing Request workflow, is no longer needed.

**Comment by cullerton** [comment]
@danfunk 

What is the context for this? Who/what is calling get_workflow? Are we creating a separate front end? Is this within current CR Connect?

get_workflow doesn't need a study, but we need somewhere to 'send' the workflow.

**Comment by cullerton** [comment]
Thoughts/Questions/Proposals:

- We can return information from a standalone workflow, but we never store anything

- Need a way to launch--and probably list, standalone workflows on the front end.
- API endpoint is /standalone_workflow/{workflow_id}

- Need to create a 'standalone' study, probably per user_id.
- Standalone workflows need to be differentiateable 
    - Is this a standalone workflow spec?
    - Is this a *type* of workflow?

- Do we want to run a cleanup after standalone workflows? Delete study?

**Comment by cullerton** [comment]
Current Plan:

- Create Workflow Specifications api endpoint /workflow-specification/{spec_id} as POST, that generates a workflow from a workflow_spec, and returns the workflow.
- Use standard path to run that workflow
- Find problems


Current snafu: TaskEventModel expects a study_id

**Comment by cullerton** [comment]
Need a way to mark a Workflow Spec as standalone. I.e., it can be run in standalone mode.

Need an endpoint to grab standalone workflows.
Add column to workflow_spec for standalone or group

**Comment by cullerton** [comment]
Frontend requirements:

- a way to look up standalone workflows
- a way to launch a standalone workflow

**Comment by calexh-sar** [comment]
@cullerton don't know of a way I can test this one. Would you pls either provide info on how I can or just close this ticket if you can confirm it has been released and working as designed.

**Comment by cullerton** [comment]
@calexh-sar I don't think we can test this yet.
I believe we are waiting on frontend code. (See my comment from Apr 23 ^^^)

What should we do with a ticket in this case?
